spec/sign-spec.ts

```diff
@@ -0,0 +1,50 @@
+import test from 'ava';
+import path from 'path';
+import { createTempDir } from '../src/temp-utils';
+import fs from 'fs-extra';
+import { createWindowsInstaller } from '../src';
+import { createTempAppDirectory } from './helpers/helpers';
+import { SignToolOptions } from '@electron/windows-sign';
+import semver from 'semver';
+
+const log = require('debug')('electron-windows-installer:spec');
+
+test.serial('creates a signtool.exe and uses it to sign', async (t): Promise<void> => {
+  if (semver.lt(process.version, '18.0.0')) {
+    log('Skipping test because Node.js < 18.0.0');
+    return;
+  }

```

Comment:
A few CI jobs are failing due to this early return since not enough assertions are being run. I haven't used Ava before, but the docs on [Assertion Planning](https://github.com/avajs/ava/blob/main/docs/03-assertions.md#assertion-planning) seem to indicate that something like this will fix it:

```suggestion
  if (semver.lt(process.version, '18.0.0')) {
    log('Skipping test because Node.js < 18.0.0');
    t.plan(0);
    return;
  }

```

Reply:
FWIW, Packager wraps its macOS signing tests inside a conditional (since they only work on macOS), instead of putting the conditional inside the test, or using test planning.

Reply:
Clever, thank you! I've added that.